components/front-page/GettingStartedSection.tsx

```diff
@@ -0,0 +1,28 @@
+import { CreateAccountActionableCard } from "components/ui/actionable/cards/CreateAccount";
+import { DepositActionableCard } from "components/ui/actionable/cards/Deposit";
+import { StartTradingActionableCard } from "components/ui/actionable/cards/StartTrading";
+
+const GettingStartedSection = () => {
+  return (
+    <>
+      <div className="w-full" data-testid="learnSection">
+        <h2 className="sm:col-span-3 text-center sm:text-start mb-6">
+          Getting Started
+        </h2>
+        <div className="flex flex-col md:flex-row gap-4">
+          <div className="flex-1">
```

Comment:
Some alignment issues on the 905px breakpoint

![Screenshot 2023-09-05 at 9 21 35 PM](https://github.com/zeitgeistpm/ui/assets/14626415/09a67418-c580-499b-b4bb-a6050c0ae286)

Reply:
Should be better in the newest build.